charts/kcp/templates/kcp-front-proxy.yaml

```diff
@@ -238,37 +233,84 @@ spec:
         image: {{ .Values.kcpFrontProxy.image }}:{{ .Values.kcpFrontProxy.tag }}
         ports:
         - containerPort: 8443
-        args:
-        - --secure-port=8443
-        - --root-kubeconfig=/etc/kcp-front-proxy/kubeconfig/root-shard.kubeconfig
-        - --tls-private-key-file=/etc/kcp-front-proxy/tls/tls.key
-        - --tls-cert-file=/etc/kcp-front-proxy/tls/tls.crt
-        - --client-ca-file=/etc/kcp-front-proxy/client/tls/ca.crt
-        - --mapping-file=/etc/kcp-front-proxy/config/path-mapping.yaml
-        - --v={{ .Values.kcpFrontProxy.v }}
-        {{- if .Values.kcpFrontProxy.profiling.enabled }}
-        - --profiler-address=localhost:{{- .Values.kcpFrontProxy.profiling.port -}}
-        {{- end }}
+        command:
+        - /busybox/sh
```

Comment:
Why?

Reply:
It's from debugging the entrypoints, will remove